.github/workflows/go.yml

```diff
@@ -1,15 +1,11 @@
 name: Go
```

Comment:
Github actions are not enabled in openshift repos, so it will not impact this repo. I think we can keep those files. It can simplify the sync operation between upstream and downstream.